h3/src/quic.rs

```diff
@@ -32,7 +32,7 @@ impl<'a, E: Error + 'a> From<E> for Box<dyn Error + 'a> {
 /// Trait representing a QUIC connection.
 pub trait Connection<B: Buf> {
     /// The type produced by `poll_accept_bidi()`
-    type BidiStream: SendStream<B> + RecvStream;
+    type BidiStream: BidiStream<B>;
```

Comment:
If we keep the previous bounds, then implementors aren't _required_ to provide splittable streams, just send and receive. If they implement BidiStream, as an opt-in, then the streams can be splittable.

But by not requiring it, we allow QUIC implementations that don't have an internal cheap way to split.